### PR TITLE
Features into development

### PR DESCRIPTION
**Pull request by IvanDerlich**
Please read comment at the end

### CODE REVIEW

**Comment by IvanDerlich** [comment]
Eslint says everything is OK but I don't know how to correct the errors accused by Stickler.

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "eol-last": "off",
+      "arrow-parens": "off"
+    },
+    "settings": {
+        "import/resolver": {
+          "node": {
+            "extensions": [""]
+          }
+        }
+      }
```
Comment (Oghenebrume50) [inline .eslintrc.json:25]: Do not edit the Eslint file see this

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "restaurant-hompage",
+  "version": "1.0.0",
+  "description": "A restaurant homepage",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (Oghenebrume50) [inline package.json:10]: configure webpack professionally like this
`
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

`

File: .eslintrc.json (hunk 1 above)
Comment (IvanDerlich) [reply .eslintrc.json]: Done

File: package.json (hunk 2 above)
Comment (IvanDerlich) [reply package.json]: Professionalized 😎

File: .eslintrc.json (hunk 1 above)
Comment (IvanDerlich) [reply .eslintrc.json]: [Done](https://github.com/IvanDerlich/restaurant-homepage/pull/3/commits/5c0e0e71380a7924cc4cc06315c49e3246c3010c)

**Comment by shloch** [comment]
Nonetheless ......add inside the readme, settup commands before merging
git cloning, webpack etc...

**Comment by IvanDerlich** [comment]
> Nonetheless ......add inside the readme, settup commands before merging
> git cloning, webpack etc...
ghhg

**Comment by IvanDerlich** [comment]
@shloch 

> Nonetheless ......add inside the readme, settup commands before merging
> git cloning, webpack etc...

[Done](https://github.com/IvanDerlich/restaurant-homepage/tree/features) :sunglasses:

**Pull request merged**